common/src/main/java/com/gregtechceu/gtceu/common/machine/multiblock/part/ItemBusPartMachine.java

```diff
@@ -68,9 +77,17 @@ protected NotifiableItemStackHandler createInventory(Object... args) {
         return new NotifiableItemStackHandler(this, getInventorySize(), io);
     }
 
+    protected NotifiableItemStackHandler createCircuitItemHandler(Object... args) {
+        return new NotifiableItemStackHandler(this, 1, IO.IN).setFilter(IntCircuitBehaviour::isIntegratedCircuit);
+    }
```

Comment:
if (arg[]s.length > 0 && args[0] instanceof IO io && io == IO.IN){
    return new NotifiableItemStackHandler(this, 1, IO.IN).setFilter(IntCircuitBehaviour::isIntegratedCircuit);
} else {
    return new NotifiableItemStackHandler(this, 0, IO.NONE);
}